Tests/PenguinStructuresTests/EitherTests.swift

```diff
@@ -0,0 +1,39 @@
+//******************************************************************************
+// Copyright 2020 Penguin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+import PenguinStructures
+
+
+class EitherTests: XCTestCase {
+  
+  func testEquality() {
```

Comment:
Should test reflexivity, symmetry, and transitivity.  Then you should factor it out into something general for Equatable in https://github.com/saeta/penguin/blob/master/Tests/PenguinStructuresTests/StdlibProtocolTests.swift.  Then the Equatable test gets used on `Collection` indices in the existing tests.  etc.